src/java.base/share/classes/java/lang/Process.java

```diff
@@ -261,7 +456,7 @@ public boolean waitFor(long timeout, TimeUnit unit)
      * when the process has terminated.
      * <p>
      * Invoking this method on {@code Process} objects returned by
-     * {@link ProcessBuilder#start} and {@link Runtime#exec} forcibly terminate
+     * {@link ProcessBuilder#start()} and {@link Runtime#exec} forcibly terminate
```

Comment:
Should the link to `Runtime#exec` be fixed in the same manner, here and elsewhere in this file?